identity-account/src/account/account.rs

```diff
@@ -48,12 +50,13 @@ pub struct Account<C = Arc<Client>>
 where
   C: SharedPtr<Client>,
 {
+  pub(crate) account_id: AccountId,
```

Comment:
Since `AccountId` is just 16 bytes, I figured it would be much more efficient to look it up once during `Account::load_identity`, and then just read the field for any future `Storage::*` calls.

  Making Storage look up the account id of a DID for every call results in a less consistent / more confusing interface than just passing `AccountId` everywhere, I think. `Storage::{key_generate, key_insert}` still need to take the `AccountId` while all others would take `IotaDID`. That is, unless we make `Storage` generate the `DID`. Also the lookup also costs a bit of performance every time, though it probably doesn't matter too much.